networks/p2p/discover/ntp.go

```diff
@@ -43,18 +44,38 @@ func (s durationSlice) Len() int           { return len(s) }
 func (s durationSlice) Less(i, j int) bool { return s[i] < s[j] }
 func (s durationSlice) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
 
+// checkClockDrift queries an NTP server for clock drifts and warns the user if
+// one large enough is detected.
+func CheckClockDriftWithRetry(url, port string) {
+	var drift time.Duration
+	for {
```

Comment:
How about adding a maximum trial limit for this loop?

Reply:
I think It must alerts the warning until the operator notices. Before this PR, we just stop the server but it would be more relaxed.